src/Initializer/InitProcedure/InitIO.cpp

```diff
@@ -0,0 +1,206 @@
+#include "Init.hpp"
+#include "InitIO.hpp"
+#include "Initializer/BasicTypedefs.hpp"
+#include <SeisSol.h>
+#include <cstring>
+#include <vector>
+#include "DynamicRupture/Misc.h"
+
+#include "Parallel/MPI.h"
+
+static void setupCheckpointing() {
+  const auto& ssp = seissol::SeisSol::main.getSeisSolParameters();
+  auto& memmng = seissol::SeisSol::main.getMemoryManager();
```

Comment:
I wouldn't use names like `ssp` and `memmng`. It is difficult to read and get semantic out of it. I would use `seissolParameters` (or `seissolParams`) and `memoryManager`.

I can see that it is in many places in the code. I just highlighted only here.